build.psm1

```diff
@@ -2058,12 +2058,14 @@ function script:Start-NativeExecution
                 $callerFile = $callerLocationParts[0]
                 $callerLine = $callerLocationParts[1]
 
+                $errorMessage = "Execution of {$sb} by ${callerFile}: line $callerLine failed with exit code $LASTEXITCODE"
```

Comment:
> ${callerFile}

Do you mean `{$callerFile}`? `${callerFile}` is the same as `$callerFile`

Reply:
![image](https://user-images.githubusercontent.com/10873629/36618836-845cff9e-18a1-11e8-875c-c82a3d66ffaf.png)

Reply:
![image](https://user-images.githubusercontent.com/10873629/36618886-afe20420-18a1-11e8-97c6-ddb0bd07ef34.png)

Reply:
OK, I missed the colon behind it. Never mind :) #Closed